components/formats-gpl/utils/writeMapAnnotationsExample.java

```diff
@@ -0,0 +1,97 @@
+import java.util.ArrayList;
+
+import ome.xml.model.MapPair;
+import loci.formats.FormatTools;
+import loci.formats.ImageWriter;
+import loci.formats.MetadataTools;
+import loci.formats.ome.OMEXMLMetadata;
+import loci.formats.services.OMEXMLServiceImpl;
+
+
+/*
+ *------------------------------------------------------------------------------
+ *  Copyright (C) 2014 University of Dundee. All rights reserved.
+ *
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ *------------------------------------------------------------------------------
+ */
+
+/**
+ * 
+ *
+ * @author Balaji Ramalingam &nbsp;&nbsp;&nbsp;&nbsp;
+ * <a href="mailto:b.ramalingam@dundee.ac.uk">b.ramalingam@dundee.ac.uk</a>
+ * @since 5.1
+ */
+public class writeMapAnnotationsExample {
+
+    public static void minimumWriterPlusMapAnnotations(String[] args) throws Exception{
+
+        if (args.length < 1) {
+            System.out.println("Please specify an output file name.");
+            System.exit(1);
+        }
+        String id = args[0];
+
+        // create blank 512x512 image
+        System.out.println("Creating random image...");
+        int w = 512, h = 512, c = 1;
+        int pixelType = FormatTools.UINT16;
+        byte[] img = new byte[w * h * c * FormatTools.getBytesPerPixel(pixelType)];
+
+        // fill with random data
+        for (int i=0; i<img.length; i++) img[i] = (byte) (256 * Math.random());
+
+        // create metadata object with minimum required metadata fields
+        System.out.println("Populating metadata...");
+
+        OMEXMLServiceImpl OMEXMLService = new OMEXMLServiceImpl();
```

Comment:
I'd suggest renaming this variable, to avoid confusion with the ```loci.formats.services.OMEXMLService``` class.  It would also be good to use ```ServiceFactory``` to construct the service, instead of directly instantiating ```OMEXMLServiceImpl``` (see https://github.com/openmicroscopy/bioformats/blob/develop/components/formats-gpl/utils/PrintTimestamps.java#L140)